apps/site/assets/ts/schedule/components/schedule-finder/ServiceSelector.tsx

```diff
@@ -74,99 +79,123 @@ const CrTableRow = ({
   schedule
 }: {
   schedule: Schedule;
-}): ReactElement<HTMLElement> => {
-  return (
-    <tr className="schedule-table__row">
-      <td className="schedule-table__td">
-        <div className="schedule-table__time">{schedule.time}</div>
-      </td>
-      <td className="schedule-table__td schedule-table__tab-num">
-        {schedule.trip.name}
-      </td>
-      <td className="schedule-table__headsign">
-        {modeIcon(schedule.route.id)}
-        {"   "}
-        {schedule.trip.headsign}
-      </td>
-    </tr>
-  );
+}): ReactElement<HTMLElement> => (
+  <tr className="schedule-table__row">
+    <td className="schedule-table__td">
+      <div className="schedule-table__time">{schedule.time}</div>
+    </td>
+    <td className="schedule-table__td schedule-table__tab-num">
+      {schedule.trip.name}
+    </td>
+    <td className="schedule-table__headsign">
+      {modeIcon(schedule.route.id)}
+      {"   "}
```

Comment:
does this triple space actually render in the HTML as three-spaces?

Reply:
Nope, fixed.